buildDockerImages.pl

```diff
@@ -193,8 +194,18 @@ sub setupForBuild {
 	runAndLog($fileout, "cp -r ./workloadConfiguration ./dockerImages/runharness/workloadConfiguration");
 }
 
+sub removeImagesAndContainers {
```

Comment:
Only one more comment; since the container removal is no longer present, this subroutine should be renamed.